recipes/excels2vensim/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "excels2vensim" %}
+{% set version = "0.10.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/excels2vensim-{{ version }}.tar.gz
+  sha256: c635b4c1906331376a4d5f92ce047477b9bb3d9960165b436e1dcd4842e2ba3f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pysd
+    - xorg-libx11  # [linux]
```

Comment:
Instead of this (which is keeping the recipe from being noarch), I think you should instead add a `yum_requirements.txt` file to the recipe, with the line `libX11` (or *maybe* `xorg-x11-server-Xorg`) since really the package wants to use the user's system X11 library--those just aren't available by default in our test environments. ([See here](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=yum_requirements#yum-requirements-txt)).

Reply:
Thanks, it worked fine :)